src/components/ExtensionMessage.js

```diff
@@ -35,15 +35,14 @@ console.log("Extensionmessage- feature experiment 1 " + Feature.extension_experi
             encyclopedias like Wikipedia.
             <br /> <br />
             To read articles with the help of Zeeguu you need to read them from
-            the Chrome Extension or by adding the texts to "Own Texts" through
```

Comment:
Should we drop the localization to Danish? 
Or shall we add the localization later? 
What do you think? 
Because at the moment we seem to have some texts localized, and some not.

Reply:
I will add it for this one - but indeed it does seem like we are not completely consistent. For example the danish version also have publish time and topics in english - and the statistics page is always in english. 

And I'm not sure this is something you know - but if I choose Danish - then every time i reload the page it switches to english again. It's only danish if I use the sidebar navigation. In the student part - not the teacher part.

Reply:
Thanks! 
I wasn't aware of the bug for the teachers.
Truth is, even for the learners it might be overkill and maybe 
we should slowly un-adopt it... Who uses our system while only speaking Danish but not English?

Reply:
good. let's deploy this for now; and we'll see about the danish at a later point!